kolibri_instant_schools_plugin/assets/src/views/about/state/store.js

```diff
@@ -0,0 +1,24 @@
+import Vuex from 'kolibri.lib.vuex';
+import * as coreStore from 'kolibri.coreVue.vuex.store';
+
+const initialState = {
+  pageName: undefined,
+  pageState: {},
```

Comment:
Do the FAQ or About page make use of `pageState`?

Reply:
Nope, will remove.